packages/lwc-language-server/package.json

```diff
@@ -48,22 +50,25 @@
     "xml2js": "^0.4.23"
   },
   "devDependencies": {
+    "@jest/reporters": "^29.5.0",
+    "@komaci/types": "^246.0.2",
+    "@lwc/old-compiler": "npm:@lwc/compiler@0.34.8",
```

Comment:
`lightning-language-server` relied upon this ancient version of the LWC compiler. It has been made a dev dependency here, allowing for the comparison of mapped metadata to original metadata.